src/components/Timeline/Event.js

```diff
@@ -1,11 +1,13 @@
 import React from 'react'
+import { connect } from 'react-redux'
 import PropTypes from 'prop-types'
 import moment from 'moment'
 import { Card, CardHeader, CardText } from 'material-ui/Card'
 import BootstrapPlaybook from './Playbook/BootstrapPlaybook'
 import Playbook from './Playbook/Playbook'
+import { LoadTextFromEvent } from '../../services/playbookService'
 
-const Event = ({ text, time, backgroundColor, incidentId, ticketId, eventTypeId, eventId, eventActions, eventTypeActions }) => {
+export const Event = ({ text, time, backgroundColor, incidentId, ticketId, eventType, eventTypeId, eventId, eventActions, eventTypeActions }) => {
```

Comment:
Can we break up this line? What line length conventions should we have for this project? (I can live with 110, but prefer to live under 80-chars :P )

Reply:
We don't really have any defined style conventions (aside from not using semicolons in JS). I wouldn't mind adding a linter rule to keep things within a certain line length (warning at first until we get everything already existing updated, then upgrading to error to prevent regression). Changing this to align arguments vertically.